server/graphql/invoice/src/mutations/prescription/update.rs

```diff
@@ -20,6 +21,7 @@ pub struct UpdateInput {
     pub status: Option<UpdatePrescriptionStatusInput>,
     pub patient_id: Option<String>,
     pub clinician_id: Option<String>,
+    pub prescription_date: Option<NaiveDate>,
```

Comment:
This should be, Option<DateTime<Utc>>, similar to filter inputs (otherwise we are sending timezoned datetime, without timezone, that will be consider as UTC in back end):

https://github.com/msupply-foundation/open-msupply/blob/aabe9159425a49fe1ee8061ad846406b8f7388c7/server/graphql/core/src/generic_filters.rs#L71-L75

Also I don't think we need `naiveDateTime` in front end utils.

I think this would need some internal wiki description at some point, i've made a task for self on trello, as a placeholder, will turn into issue or wiki TODo

Reply:
This was going to be date, with no time (assumed midnight) but timezone are an issue good point.
I was copying another pattern that did you naive date, I think because there wasn't a time component at all.

Definitely can remove `naiveDateTime` in front end utils, I put that in earlier then didn't end up using it and forgot to remove thanks for picking this up!

Reply:
[1ca5619](https://github.com/msupply-foundation/open-msupply/pull/4788/commits/1ca56196bc1ddf1d5aaf0b726445053ec4c03c4a)